src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -78,10 +91,78 @@ Globalnet currently relies on `kube-proxy` and thus will only work with deployme
 Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
 
 This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for Globalnet. The Lighthouse
-controller adds the service's globalIp to the `ServiceImport` object that is distributed to all clusters. The [lighthouse
-plugin](https://github.com/submariner-io/lighthouse/tree/devel/plugin/lighthouse) then uses the Service's globalIp when replying to DNS
-queries for the Service.
+controller uses pod's globalIp instead of podIp when creating `EndpointSlice` objects to be distributed to other clusters.
```

Comment:
Can you please clarify this further. Like when does LH use Pod globalIP?

Reply:
It is used in case of HeadlessService. For headless services DNS returns list of PodIPs backing a service, so in case of globalnet we need to return pod globalIPs. I'll modify this to further clarify the difference between headless and clusterIP.